tokens/src/core/alias/size.json

```diff
@@ -1,11 +1,11 @@
 {
   "size": {
     "border": {
-      "width": { "value": "1px", "type": "dimension", "source": "$border-width" },
+      "width": { "value": "1px", "type": "dimension", "source": "$border-width", "description": "Border width value." },
       "radius": {
-        "base": { "value": "6px", "type": "dimension", "source": "$border-radius" },
-        "lg": { "value": "7px", "type": "dimension", "source": "$border-radius-lg" },
-        "sm": { "value": "4px", "type": "dimension", "source": "$border-radius-sm" }
+        "base": { "value": "6px", "type": "dimension", "source": "$border-radius", "description": "Border radius basic." },
+        "lg": { "value": "7px", "type": "dimension", "source": "$border-radius-lg", "description": "Border radius large." },
+        "sm": { "value": "4px", "type": "dimension", "source": "$border-radius-sm", "description": "Border radius small." }
```

Comment:
Another place to decide on naming conventions. My gut feeling is "Large border radius" feels better than "Border radius large.", but I'd like to hear more thoughts.

Reply:
Changed to `Large border radius` and for the second one also